docs/kola/external-tests.md

```diff
@@ -235,6 +236,9 @@ at least the specified amount of memory is used. On QEMU, this is equivalent to
 the `--memory` argument to `qemuexec`. This is currently only enforced on
 `qemu-unpriv`.
 
+The `additionalNics` key has the same semantics as the `--secondary-nics` argument
```

Comment:
Hmm, I wonder if we should rename the switch to `--additional-nics` to be consistent. I'm not sure how prevalent the use of the previous switch is at the moment though.

/cc @RamLavi

Reply:
yeah. We should find consistency either way.

Reply:
Agree, I will create another PR to change all related `secondary-nics` to `additional-nics` after this is merged. Let's focus on `additionalNics` feature first.

Reply:
Sounds good to me.